### ISSUE TITLE
Peer Review from Guanshu Tao

### ISSUE THREAD

**Issue by GuanshuTao** [UBC-MDS/covid-19-cases-vs-tests-analysis#79]
* **Topic :** 
This topic is highly related to the pandamic and I'm highly attracted to this topic.

* **Code :**
I looked into the code files and the docstrings are succint and clear. All code is written neatly with direct information and working well.

* **EDA Report :**
This report is well-organized and each step is explained clearly with reasonable logic. Perhaps more visualization can be given instead of some of the tables. I think I would also like to see graphs to interpret new cases and new tests for Canada/USA! I like the ridge plot very much and it is a nice report overall! I'm not sure if the eda folder should be deleted or not but I enjoyed reading it and hope you can keep it in the repository.

* **Final Report :**
Latex is a nice tool to render experiment reports and you also added hyperlinks to the table of contents so it is very convenient to read.  I think it would be great if you cover some keypoint especially the plot from the EDA report into the final report. The line graph and ridge plot provides suppoting information to your final report they might be helpful to build the report more intergrated. 

* **General :**
This project is well executed with a interesting topic and I think it is great work! A followed up topic comes to my mind: prediction on the trend of difference of Convid-19 response ratio between Canada and USA in 2021. Perhaps it can provides you more ideas about future work. Overall, I really liked to read through your works!

Good luck on the rest of project and stay healthy!

**Comment by fatse** [comment]
@GuanshuTao thank you very much for reviewing our project thoroughly and providing your constructive feedback!

**Comment by tanmaysharma19** [comment]
Thank you for taking the time to review and providing your feedback @GuanshuTao. Appreciate it!

The EDA folder is still available under https://github.com/UBC-MDS/covid-19-cases-vs-tests-analysis/releases/tag/0.1.0.

The folder was removed from the final milestone releases with the intention of only showing the most relevant code and analysis regarding our statistical question. The same was discussed with Tiffany and we got a go-ahead.
